Upgrade nispor dependency to 1.2.6

cathay4t opened the pull request:
The nispor 1.2.6 fixed the linux bridge bug:

    https://github.com/little-dude/netlink/pull/247

Which is important to nmstate

ffmancera reviewed (changes requested):
The netlink crates dependencies must be updated too, thanks!

cathay4t committed:
Upgrade nispor dependency to 1.2.6

The nispor 1.2.6 fixed the linux bridge bug:

    https://github.com/little-dude/netlink/pull/247

Which is important to nmstate

Signed-off-by: Gris Ge <fge@redhat.com>

cathay4t commented:
> The netlink crates dependencies must be updated too, thanks!

Fixed. Thanks!

cathay4t commented:
Dropping. As we might need nispor 1.2.7 for other fixes.
